src/main_menu.cpp

```diff
@@ -411,15 +589,19 @@ bool main_menu::opening_screen()
 
     load_char_templates();
 
-    ctxt.register_cardinal();
+    ctxt.register_directions();
```

Comment:
Why directions? I see no use of LEFTUP, LEFTDOWN, RIGHTUP or RIGHTDOWN action

Reply:
Yep, that was a slip-up from testing. Cheers.

```suggestion
    ctxt.register_cardinal();
```